thoth/storages/graph/postgres.py

```diff
@@ -3375,6 +3392,50 @@ def get_python_package_versions_count_all(
 
             return query.count()
 
+    def get_multi_values_key_python_package_version_metadata(
```

Comment:
Can this method be private?

Reply:
And also `MULTI_VALUE_KEY_PYTHON_PACKAGE_METADATA_MAP`.

Reply:
Updated, thanks :)